src/mayur/assignment15/SwitchBank.java

```diff
@@ -0,0 +1,128 @@
+package mayur.assignment15;
+
+public class SwitchBank {
+	int iBalance=25000;
+	int hBalance=20000;
+	int bBalance=10000;
+	int sBalance=15000;
+	int iCreditCount = 0, hCreditCount = 0, bCreditCount = 0,sCreditCount=0, totalCreditCount = 0;
+	int iDebitCount = 0, hDebitCount = 0, bDebitCount = 0,sDebitCount=0, totalDebitCount = 0;
+
+	void displayDebitCredit(String bank,String op,int amt) {
+		switch(op) {
+		case "debit":
+			switch(bank) {
+			case "ICICI" :
+				if(iBalance>=amt) {
```

Comment:
instead of writing multi-time same code block you can write debit() method which accept the two arguments and you can simply call that method